### PR TITLE
[PREVIEW] Read messages from the queue

### PR DESCRIPTION
**Pull request by rkondratowicz**

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -12,6 +12,10 @@ spring:
 
 bulk-scan-processor-url: ${BULK_SCAN_PROCESSOR_URL:}
 
+queue:
+  conn-string: ${QUEUE_CONNECTION_STRING:}
```
Comment (timja) [inline src/main/resources/application.yaml]: how about: connection-string
you're doing the longer version in the env variable anyway, may as well be consistent

Comment (nitinprabhu) [inline src/main/resources/application.yaml]: Is terraform config part of separate PR ?

Comment (rkondratowicz) [reply src/main/resources/application.yaml]: done

Comment (rkondratowicz) [reply src/main/resources/application.yaml]: it's actually already on master, thanks to @timja 
https://github.com/hmcts/bulk-scan-orchestrator/blob/master/infrastructure/main.tf#L39

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -12,6 +12,10 @@ spring:
 
 bulk-scan-processor-url: ${BULK_SCAN_PROCESSOR_URL:}
 
+queue:
+  connection-string: ${QUEUE_CONNECTION_STRING:}
+  read-interval: ${QUEUE_READ_INTERVAL:30000} # in ms
```
Comment (luigibk) [inline src/main/resources/application.yaml]: Won't we need to rename all the connection string variables if we need to add other queues?

Comment (rkondratowicz) [reply src/main/resources/application.yaml]: **IF** we do, then yes :)

**Pull request merged**